source/codegen/metadata/niswitch/functions.py

```diff
@@ -962,7 +997,8 @@
         'returns': 'ViStatus'
     },
     'Init': {
-        'cname': 'niSwitch_init ',
```

Comment:
This is just a funny callout that this used to have a space in it which nothing complained about. You can see it reflected in some of the changes in the generated files where it got rid of the space like the old version of
```  return niSwitch_init (resourceName, idQuery, resetDevice, vi);```